sdk/keyvault/Azure.Security.KeyVault.Keys/api/Azure.Security.KeyVault.Keys.netstandard2.0.cs

```diff
@@ -13,8 +13,10 @@ public partial class CreateKeyOptions
         public CreateKeyOptions() { }
         public bool? Enabled { get { throw null; } set { } }
         public System.DateTimeOffset? ExpiresOn { get { throw null; } set { } }
+        public bool? Exportable { get { throw null; } set { } }
         public System.Collections.Generic.IList<Azure.Security.KeyVault.Keys.KeyOperation> KeyOperations { get { throw null; } }
         public System.DateTimeOffset? NotBefore { get { throw null; } set { } }
+        public Azure.Security.KeyVault.Keys.KeyReleasePolicy ReleasePolicy { get { throw null; } set { } }
```

Comment:
Will other types of keys not use a `ReleasePolicy`? Also, I thought we were renaming `KeyVaultReleasePolicy` to `ReleasePolicy`, did you mean to rename the parameter but not the class itself?

Reply:
Yes, just the property should be `ReleasePolicy`. The model/class is `KeyReleasePolicy`. That was in the issues I opened as well.